server/app/src/main/java/com/ams/app/controller/StudentController.java

```diff
@@ -0,0 +1,84 @@
+package com.ams.app.controller;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+import com.ams.app.dto.Requestdto;
+import com.ams.app.dto.Statsresponsedto;
+import com.ams.app.dto.Studentdashrequestdto;
+import com.ams.app.dto.Studentdashresponsedto;
+import com.ams.app.dto.Studentdto;
+import com.ams.app.dto.SubjectStatsresponsedto;
+import com.ams.app.service.StudentService;
+
+@RestController
+@RequestMapping("/student")
+public class StudentController {
+
+
+    @Autowired
+    StudentService studentService;
```

Comment:
Don't use Autowired Annotation, Instead we can use `@RequiredArgsConstructor(onConstructor = @__({@Autowired}))`

Reply:
modified